CUDADataFormats/SiPixelCluster/interface/SiPixelClustersCUDA.h

```diff
@@ -49,11 +49,11 @@ class SiPixelClustersCUDA {
     friend SiPixelClustersCUDA;
 
   private:
-    uint32_t const *moduleStart_ = nullptr;
-    int32_t  const *clus_ = nullptr;
-    uint32_t const *clusInModule_ = nullptr;
-    uint32_t const *moduleId_ = nullptr;
-    uint32_t const *clusModuleStart_ = nullptr;
+    uint32_t const *moduleStart_;
```

Comment:
why do you remove the `nullptr` initialisation ?

Reply:
Because those lead to constructor being non-trivial (since it would do some initialization).

Reply:
```cpp
#include <iostream>
class Xyz {
  int bar = 0;
};
int main(void) {
  std::cout << "Xyz " << std::is_trivially_constructible_v<Xyz> << std::endl;
  return 0;
}
```
gives
```
Xyz 0
```

Reply:
Yes, of course sorry (trivial really means it does nothing...).